test/fixtures/test-runner/bail/error.js

```diff
@@ -0,0 +1,12 @@
+const assert = require('assert');
+const common = require('../../../common');
+
+const test = require('node:test');
+
+test('keep error', (t) => {
+    assert.strictEqual(0, 1);
```

Comment:
Two space indentation here and on line 11 (unfortunately, we don't lint the fixtures).